docs/release-notes/8.7.asciidoc

```diff
@@ -0,0 +1,83 @@
+* Adds buttons to the Rules page that allow you to view only enabled or disabled rules ({pull}150153[#150153]).
+* The *Related alerts by process ancestry* section of the alert details flyout is now GA ({pull}152011[#152011]).
+* Adds the option to suppress custom query rule alerts during a specific time window. Duplicate alerts within that time window will be grouped ({pull}148868[#148868]).
+* Reduces alert creation errors by stopping the detection engine from writing non-ECS-compliant fields to alerts from source events ({pull}147628[#147628]).
+* Simplifies the interface for navigating from a rule's details page back to the Rules page ({pull}147357[#147357]).
+* Allows you to resize the Rule preview panel ({pull}147351[#147351]).
+* Improves the Bulk Edit API by adding a `skipped` property to rules that weren't updated, and updates the toast message to show which were skipped ({pull}147345[#147345]).
+* Allows placeholder fields in Osquery queries ({pull}146598[#146598]).
+* Allows the Rules table state to persist even after you refresh or navigate to another page ({pull}145111[#145111]).
+* Improves data fetch performance throughout {elastic-sec}, especially for deployments with large indices and many integrations ({pull}142904[#142904]).
+* Fixes a bug that could cause your cursor to jump to the end of the text field when editing a rule action message ({pull}150823[#150823]).
+* Fixes a bug that could result in incorrect links to machine learning jobs from search results ({pull}150881[#150881]).
+* Fixes a bug that caused a fade in and out effect on alert descriptions ({pull}150998[#150998]).
+* Fixes a bug that caused the Alerts page to default to the wrong chart type ({pull}151073[#151073]).
+* Fixes a bug that could hide some shared exception lists when you changed the number of rows in the exceptions lists view ({pull}151393[#151393]).
+* Removes a blank option from the *Field* browser in the Add rule exception flyout ({pull}151398[#151398]).
+* Fixes a UI text bug that conflated Endpoint exceptions with regular rule exceptions ({pull}151532[#151532]).
+* Fixes a bug that could cause an unnecessary warning to display in the Add rule exception flyout ({pull}151570[#151570]).
+* Fixes a bug with the empty state that appears when your exception lists search yields no results ({pull}151530[#151530]).
+* Fixes a bug that sometimes prevented a Timeline from saving when it was created using the *Investigate in timeline* action on an alert ({pull}151616[#151616]).
+* Fixes a bug that could cause unnecessary validation errors in text entry fields in the Add rule exception flyout ({pull}151654[#151654]).
+* Fixes a bug that caused some module names to be partially hidden on the Overview dashboard ({pull}151843[#151843]).
+* Fixes a visual bug that affected empty rule previews ({pull}151869[#151869]).
+* Fixes a bug that could cause alert details to incorrectly show installed integrations as not installed ({pull}152055[#152055]).
```

Comment:
I think by now we should be using the term `rule` instead of `alert` everywhere, at least in the app we renamed it a long time ago. Do we still call detection rules "alerts" in the docs?

Another thing is that the related integrations UI is displayed not on the Rule Details page, but also in the Rules table in each row. So I'd reformulate this a bit:

`Fixes a bug that could cause rule's related integrations that are installed to be incorrectly shown as not installed in the Rules table and on the rule details page`

We can also add https://github.com/elastic/kibana/pull/149646 to the same bullet point -- it addresses the same bug.

Reply:
Thanks for the review Georgii! We definitely don't use alerts to refer to detection rules, so I think you're right about this being incorrect. Thank you for the excellent edit.